tests/internal/src/Example.cpp

```diff
@@ -6,20 +6,14 @@
 #include "Example.hpp"
```

Comment:
Is the ``.clang-format`` file missing? I didn't see it.

Reply:
I was using the one given by googlebenchmark.  Couple things went wrong: moving includes and the lambda's were ugly.